client/lib/reader-site-store/index.js

```diff
@@ -30,6 +30,10 @@ function setSite( attributes ) {
 		attributes.state = State.COMPLETE;
 	}
 
+	if ( attributes.meta && attributes.meta.links ) {
```

Comment:
could use lodash's get here and make things a bit simpler https://lodash.com/docs#get